packages/ember/package.json

```diff
@@ -29,7 +29,7 @@
     "test:all": "node ./scripts/run-CI-tests.js"
   },
   "dependencies": {
-    "@embroider/macros": "~0.47.2",
+    "@embroider/macros": "1.8.3",
```

Comment:
Should this be `^1.8.3` similar to what `@ember-data/model` has as dependency? https://github.com/emberjs/data/blob/master/packages/model/package.json#L30

Reply:
It's fine leaving it exact, we have a lock anyway.